go.mod

```diff
@@ -18,9 +17,75 @@ require (
 	github.com/onsi/gomega v1.10.2
 	github.com/segmentio/ksuid v1.0.3
 	go.uber.org/zap v1.10.0
+	golang.org/x/exp v0.0.0-20221031165847-c99f073a8326
```

Comment:
Please see comment about not using `golang.org/x/exp/slices` in `modrulestore.go`.